src/ott/utils.py

```diff
@@ -59,3 +60,56 @@ def is_jax_array(obj: Any) -> bool:
     # https://jax.readthedocs.io/en/latest/jax_array_migration.html
     return isinstance(obj, (jax.Array, jnp.DeviceArray))
   return isinstance(obj, jnp.DeviceArray)
+
+
+def default_progress_fn(
+    status: Tuple[np.ndarray, np.ndarray, np.ndarray, NamedTuple], *args
```

Comment:
In the docs, you mention `jax.Array`, but the signature expects `np.ndarray`.

Reply:
Yup, that's fixed now, these are indeed `np.ndarray`, except the last argument which is a `sinkhorn.SinkhornState`. But FYI I cannot named that type here because importing it creates a cyclic import (`ott/utils.py` is included in many places).